### DISCUSSION TITLE
Functions log are stuck at `Awaiting new requests…`

### DISCUSSION THREAD

**Discussion by shunkakinoki** [vercel/vercel#7560] [Help]
Hi, love the vercel platform, I can't imagine a life without it any longer.

Regarding the automatic serverless functions hosted with vercel, 
although I can confirm they are working perfectly with `getServerSideProps`, 
the logs are stuck at `Awaiting new requests…`

Likely a bug, and wish they will be fixed soon!

**Comment by stefanbajic94** [comment]
I'm experiencing the same issue, the logs show no requests coming in but I do get responses from serverless functions.

**Comment by janzheng** [comment]
this issue seems to still happen, even with `now logs [deployment URL] -f`

**Comment by ghost** [comment]
I had the same problem. `vercel logs [URL] -f` show only building logs.
